### PR TITLE
5911 create dashboard page

### PR DESCRIPTION
**Pull request by emitche**
- Create dashboard page
- Create link in menu

### CODE REVIEW

File: app/controllers/admin/dashboard_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Admin::DashboardController < Admin::AdminController
+  def dashboard
+    authorize Project
```
Comment (emitche) [inline app/controllers/admin/dashboard_controller.rb]: Project will be used by 3 out of the 4 features that are a part of the dashboard. But I can remove this until it is needed, if you think that is best.

File: app/controllers/admin/dashboard_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Admin::DashboardController < Admin::AdminController
+  def dashboard
```
Comment (emitche) [inline app/controllers/admin/dashboard_controller.rb]: When pairing with linesarefuzzy, Fuzzy requested that this action not be index or show, but instead a custom action named dashboard.

**Comment by emitche** [comment]
Page title is missing and needs to be added.

File: app/assets/stylesheets/admin/dashboard/_dashboard.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.admin-dashboard {
+  .content {
+    .row {
+      margin-bottom: 1em;
+    }
+
+    section {
+      section {
```
Comment (hooverlunch) [inline app/assets/stylesheets/admin/dashboard/_dashboard.scss]: This seems like an odd pattern and perhaps a bit brittle. Can you just use a class on the inner section perhaps?

File: app/controllers/admin/dashboard_controller.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+    authorize Project
+    authorize Person
```
Comment (hooverlunch) [inline app/controllers/admin/dashboard_controller.rb]: We should talk about this authz stuff. Not quite right.

File: app/controllers/admin/dashboard_controller.rb (hunk 2 above)
Comment (hooverlunch) [inline app/controllers/admin/dashboard_controller.rb]: Hmm, I'm curious as to why...

**Comment by emitche** [comment]
Per @rgalusca, the heading should be in line with the beginning of the menu.

**Comment by emitche** [comment]
The page can be accessed at admin/dashboard. This feature is just the bones for the dashboard.

**Pull request merged**